apps/03TicTacToe.js

```diff
@@ -25,22 +30,40 @@ function printBoard() {
 
 function horizontalWin() {
   // Your code here
+  if ((board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) || (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn) || (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn)){
+      return true;
+  }
+  
 }
 
 function verticalWin() {
   // Your code here
+   if ((board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn)  ||  (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn) || (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn)){
+      return true;
+  }
+
 }
 
 function diagonalWin() {
   // Your code here
+   if ((board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn) || (board[2][0] === playerTurn && board[1][1] === playerTurn && board[0][2] === playerTurn)){
+      return true;
+  }
 }
 
 function checkForWin() {
   // Your code here
+  if (horizontalWin(), verticalWin(), diagonalWin()) {
+    return true;
+    console.log('Player ' + playerTurn + ' Won!');
+  }
 }
 
 function ticTacToe(row, column) {
   // Your code here
+  board[row][column] = playerTurn;
+  toggleTurn();
```

Comment:
checkForWin() is not called anywhere in your app, so the function will never run and a win will never be declared.